<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Investigating dead ends in the vessel network</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonAmFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Input network file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAmFile">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>451</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDeadends">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Detect dead ends</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>491</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>371</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOutputFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Amira file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutputFile">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>451</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonImageFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Input image file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTiffFile">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>451</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNdead">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonEvaluate">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Evaluate dead ends</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxVoxel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>160</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_zvoxel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_z">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_xvoxel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_x">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_yvoxel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_y">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_voxel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Voxel Size (um)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxSphere">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEditRadius">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditZ">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditY">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelSphere">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Centre</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelR">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Radius</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditX">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSphere">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Select sphere</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_y_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_x_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_z_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_um">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>(um)</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonEvaluateAll">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Evaluate network</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSaveFile">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save dead-end file</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rlimit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Max radius for intensity calc.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRlimit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>270</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonAmFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>amFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>21</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonImageFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>tiffFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonOutputFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>outputFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonDeadends</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>detectDeadends()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonEvaluate</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>evaluate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxSphere</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>optionSphere()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_xvoxel</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>voxelChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>773</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_yvoxel</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>voxelChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>674</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_zvoxel</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>voxelChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>480</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>582</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonEvaluateAll</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>getAllIntensities()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSaveFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveDeadendFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>outputFileSelecter()</slot>
  <slot>amFileSelecter()</slot>
  <slot>tiffFileSelecter()</slot>
  <slot>detectDeadends()</slot>
  <slot>evaluate()</slot>
  <slot>optionSphere()</slot>
  <slot>voxelChanged()</slot>
  <slot>getAllIntensities()</slot>
  <slot>saveDeadendFile()</slot>
 </slots>
</ui>
